Assets/BossRoom/Scripts/Client/UI/ConnectionStatusMessageUIManager.cs

```diff
@@ -38,13 +38,17 @@ void OnConnectStatus(ConnectStatus status)
                     PopupPanel.ShowPopupPanel("Connection Failed", "The Host is full and cannot accept any additional connections.");
                     break;
                 case ConnectStatus.Success:
+                    PopupPanel.ShowPopupPanel("Success!", "Joining Now");
```

Comment:
don't all our popup panels have an "ok" button to click? We shouldn't have to click "ok" on a connection success.

Reply:
good point, I'll add something that auto-closes after x seconds, or maybe wait until a scene load. I'll look into which one feels better for this

Reply:
Went with a manual close instead, I think it makes more sense. For this one, I call it when a new scene is loaded, and for the reconnect popup, I call it when a new connection satus message is received